src/js/api/setup-steps.js

```diff
@@ -2,13 +2,22 @@ import { PLAYLIST_LOADED, ERROR } from 'events/events';
 import PlaylistLoader from 'playlist/loader';
 import Playlist, { filterPlaylist, validatePlaylist, wrapPlaylistIndex } from 'playlist/playlist';
 import ScriptLoader from 'utils/scriptloader';
-import { composePlayerError, PlayerError,
-    SETUP_ERROR_LOADING_PLAYLIST, SETUP_ERROR_LOADING_PROVIDER,
+import { composePlayerError, convertToPlayerError, PlayerError,
+    SETUP_ERROR_LOADING_PLAYLIST, SETUP_ERROR_LOADING_PROVIDER, SETUP_ERROR_LIVESYNCDURATION,
     ERROR_LOADING_TRANSLATIONS, ERROR_LOADING_TRANSLATIONS_EMPTY_RESPONSE } from 'api/errors';
 import { getCustomLocalization, isLocalizationComplete, loadJsonTranslation, isTranslationAvailable, applyTranslation } from 'utils/language';
 import { bundleContainsProviders } from 'api/core-loader';
 import pluginsPromise from 'plugins/plugins';
 
+export const validateConfig = (model) => {
+    return new Promise((accept, reject) => {
+        if (model.attributes.liveSyncDuration > 45) {
+            accept(composePlayerError(new Error(), SETUP_ERROR_LIVESYNCDURATION));
```

Comment:
Should this be a reject?

Reply:
Using reject causes a technical error so the player crashes. Using accept/resolve we get the warning in console without the crash.
<img width="1766" alt="Screen Shot 2022-02-08 at 2 28 17 PM" src="https://user-images.githubusercontent.com/95260843/153086659-97267433-d769-481c-99c8-3d63219aab51.png">